src/store/navigation/navigation-action.ts

```diff
@@ -11,11 +11,12 @@ import { dataExplorerActions } from "../data-explorer/data-explorer-action";
 import { PROJECT_PANEL_ID } from "../../views/project-panel/project-panel";
 import { RootState } from "../store";
 import { Resource, ResourceKind } from "../../models/resource";
+import { goToProject, goToCollection } from "../../common/actions";
 
 export const getResourceUrl = <T extends Resource>(resource: T): string => {
     switch (resource.kind) {
-        case ResourceKind.Project: return `/projects/${resource.uuid}`;
-        case ResourceKind.Collection: return `/collections/${resource.uuid}`;
+        case ResourceKind.Project: return goToProject(resource.uuid);
```

Comment:
This function is used around other modules and is associated with Resource so it could be moved to models/urls.ts for example.

Reply:
@burzomir here we are using only case for Project.